libs/prelude/Prelude.idr

```diff
@@ -1509,9 +1510,32 @@ prim__putChar : Char -> (1 x : %World) -> IORes ()
 %foreign "C:getchar,libc 6"
 %extern prim__getChar : (1 x : %World) -> IORes Char
 
+read_line_js : String
```

Comment:
move this to a separate module and import it. 

why are there two read_line_js?